CIP-?/README.md

```diff
@@ -0,0 +1,99 @@
+
+4. **User Selection:**
+
+   - The user selects one of the options presented in the prompt. If they choose to resolve with Project A, the Project A's resolver is used; if they choose the other project, the respective resolver is used.
+
+5. **Default Project:**
+
+   - Users may have the option to set a default project for names with a colliding suffix in their wallet settings. This default project will be used for resolution unless changed by the user during the resolution prompt.
+   - Without user specification, there should always be an option prompting to user in case of domain name conflict.
+
+## Rationale: how does this CIP achieve its goals?
+
+This proposal embraces the decentralization property of a blockchain where it welcomes multiple domain name service that exists in the market without user experience competition at the infrastructure level. It ensures a user-friendly experience and allows users to make informed decisions in case of naming conflicts. It accommodates multiple projects with the same suffix in wallets while avoiding disputes and confusion, fostering the integration process of wallet and domain service.
+
+### Path to Active
+
+#### Acceptance Criteria
+
+- At least 3 of wallets listed below agree with the approach
+
+  - Begin <https://begin.is/>
+  - Eternl <https://eternl.io/>
+  - Flint <https://flint-wallet.com/>
+  - GeroWallet <https://www.gerowallet.io/>
+  - Lace <https://www.lace.io/>
+  - Nami <https://namiwallet.io/>
+  - NuFi <https://nu.fi/>
+  - RayWallet <https://raywallet.io/>
+  - Yoroi <https://yoroi-wallet.com/>
+
+  > Initial list of wallet adapted from CIP30, other wallet providers could to request to add to the list.
+
+#### Implementation Plan
+
+- Every participating Cardano domain service provider provides an address resolver SDK.
+- Every participating Cardano domain service provider provides either a desired prefix or suffix.
+- Wallet providers to execute and integrate with resolving address, domain service project to provide assistance.
+
+#### Participating Cardano Domain Service
+
+| Domain Service Project     | Prefix | Suffix | Link to Resolver SDK Repo                     |
+| -------------------------- | ------ | ------ | --------------------------------------------- |
+| Cardano Name Service (CNS) | N/A    | `.ada` | https://github.com/cns-space/cns-resolver-sdk |
+|                            |        |        |                                               |
+|                            |        |        |                                               |
+
+## CIP Extendibility: Domain Information Resolving
+
+This approach could be applied in resolving other information as well in case of conflict. When there is no other specific CIP covering the particular domain information resolving mechanism, the similar approach with this CIP would by default covering the particular scope.
+
```

Comment:
There is no such standard section here.  I believe your "approach" belongs Rationale because it explains why you've chosen to define the CIP so it can relate well with other CIPs coming along in the future (?), so it could be moved there.